docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc

```diff
@@ -2748,38 +2731,10 @@ reload
 the _elytron_ subsystem. If these dependencies are not resolved before
 disabling it, you will see errors when starting WildFly.
 
-[[enable-and-disable-the-security-subsystem]]
```

Comment:
Just wondering if we should also remove the section about enabling/disabling the elytron subsystem too since it's present by default and shouldn't be removed.